experimental/dds/tree2/src/test/feature-libraries/sequence-field/rebase.spec.ts

```diff
@@ -937,6 +940,27 @@ describe("SequenceField - Rebase", () => {
 		assert.deepEqual(rebased, expected);
 	});
 
+	it("insert ↷ [delete, insert]", () => {
+		// Because B does not have lineage for A, we should B's insertion's tiebreak policy
+		// and considered the cell it inserts into to be before the cell emptied by A
```

Comment:
```suggestion
		// Because B does not have lineage for A, we should use B's insertion's tiebreak policy
		// and consider the cell it inserts into to be before the cell emptied by A
```